### DISCUSSION TITLE
Maintain indication that logger row is clickable

### DISCUSSION THREAD

**Discussion by partingscientist** [uBlockOrigin/uBlock-issues#3092] [Ideas] [answered]
With changes in [1.55.1b16](https://github.com/gorhill/uBlock/commit/869a653fdf2dd4e5b0151530bd58e1059d1318d2#diff-a7c03083a8136ebd2a95281c2378377696730b665886490417d343e5ad9d3034L342), hovering over a logger row will no longer change user's cursor into `zoom-in`. Keeping this behaviour (or changing it into something similar e.g. `pointer`) might be beneficial to indicate that each logger row is still clickable.

**Comment by gorhill** [comment, marked as answer]
I prefer default because then the text cursor shows when hovering text, and it's easier to select text with the text cursor.

**Comment by ghost** [comment]
Ah, previously clicking did not work in cells with domain and URL, it works now. I think it's fine.
